components/appbar/overview.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Overview
+page_title: AppBar Overview
+description: Overview of the AppBar component for Blazor.
+slug: appbar-overview
+tags: telerik,blazor,appbar,navbar
+published: True
+position: 0
+---
+
+# Blazor AppBar Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/appbar" target="_blank">Blazor AppBar component</a> helps you build navigation bars for your application seemingly. This article explains the available features.
+
+## Creating Blazor AppBar
+
+1. Add the `<TelerikAppBar>` tag to a Razor file.
+1. Use the `<AppBarSection>` child tag to add content to the AppBar component.
+1. (optional) Use [spacers or separators](#content-dividers) to add visual distinction between the sections in the AppBar.
+
+>caption Basic configuration of the Telerik AppBar
+
+````CSHTML
+<TelerikAppBar>
+    <AppBarSection>
+        <span>Company Logo</span>
+    </AppBarSection>
+
+    <AppBarSpacer></AppBarSpacer>
+
+    <AppBarSection>
+        <span>Our Products</span>
+    </AppBarSection>
+
+    <AppBarSeparator></AppBarSeparator>
+
+    <AppBarSection>
+        <span>Our Mission</span>
+    </AppBarSection>
+
+    <AppBarSeparator></AppBarSeparator>
+
+    <AppBarSection>
+        <span>Contact Us</span>
+    </AppBarSection>
+
+    <AppBarSpacer></AppBarSpacer>
+
+    <AppBarSection>
+        <TelerikSvgIcon Icon="@SvgIcon.User"></TelerikSvgIcon>
+    </AppBarSection>
+
+    <AppBarSeparator></AppBarSeparator>
+
+    <AppBarSection>
+        <TelerikSvgIcon Icon="@SvgIcon.Logout"></TelerikSvgIcon>
+    </AppBarSection>
+</TelerikAppBar>
+````
+
+## AppBar Sections
+
+Use the AppBar Sections to render arbitrary HTML content to match the UI and UX needs of your application. [Read more about the Blazor AppBar sections]({%slug appbar-sections%}).
```

Comment:
Read more about... - use "..." as a part of the anchor link instead of "." This comment is valid for all sections with features.
```suggestion
Use the AppBar Sections to render arbitrary HTML content to match the UI and UX needs of your application. [Read more about the Blazor AppBar sections...]({%slug appbar-sections%})
```